src/maps/london/move_good.ts

```diff
@@ -7,9 +7,11 @@ import { OnRollData } from "../../engine/state/roll";
 import { Good, goodToString } from "../../engine/state/good";
 import { Random } from "../../engine/game/random";
 import { CoordinatesZod } from "../../utils/coordinates";
+import { City } from "../../engine/map/city";
+import { Space } from "../../engine/map/grid";
 
 export const LondonMoveData = MoveData.extend({
-  city: CoordinatesZod,
+  city: CoordinatesZod.optional(),
```

Comment:
Could you add a comment about what it means when this is undefined? It took me a minute to parse that out.